packages/genesys-spark-components/src/components/stable/gux-button/gux-button.scss

```diff
@@ -68,7 +68,6 @@ button {
   }
 
   &[disabled] {
-    pointer-events: none;
```

Comment:
Removed in order for the tooltip to show when the button is `disabled`. Is there any context in which this is would be required. I dont think emits events as long as we have `pointer-events:none` on the host.